frontend/src/app/modules/auth/sign-up/sign-up.component.sass

```diff
@@ -0,0 +1,176 @@
+@import src\styles\colors.sass
+
+@font-face
+    font-family: Barlow
+    src: url('~src/assets/fonts/Barlow-Medium.ttf')
+
+@mixin background-icon
+    position: absolute
+    width: 87px
+    height: 87px
+    border-radius: 50%
+    background: $time-color
+    opacity: 0.4
+
+::ng-deep *
+    margin:0
+
+::placeholder
+    color: $gray-200
+
+
+.sign-up
+    background-color: $gray-100
+    height: 100vh
+    width: 100vw
+    position: fixed
+    top:0
+    left:0
+    font-family: 'Barlow'
+    font-style: normal
+
+.registration-form
+    position: absolute
+    left: 15%
+    top: 50px
+    display: flex
+    flex-direction: column
+    width: 18%
+    height: 90%
+    h1
+        font-size: 24px
+        line-height: 146%
+        color: $main-color
+        margin-bottom: 15px
+
+.image-section
+    margin-left: 50%
+    height: 100%
+    width: 50%
+    display: flex
+    justify-content: center
+    align-items: center
+    background: repeating-linear-gradient(75deg,$accent-color,$accent-color 1px, rgba(1,1 ,1 , 0) 1px, rgba(1,1 ,1 , 0) 190px), repeating-linear-gradient(166deg,$accent-color,$accent-color 1px, rgba(1,1 ,1 , 0) 1px, rgba(1,1 ,1 , 0) 190px)
+    background-color: $main-color
+    h1
+        color: $white
+        font-size: 330%
+        line-height: 127.5%
+        text-align: center
+        text-shadow: -2px 2px 1px $accent-color
+    .car-background
+        @include background-icon
+        right: 30%
+        top: 46px
+    .car
+        position: absolute
```

Comment:
Have you tried to change width of your browser window?

Reply:
Of course, there is a screenshot in the description